### PR TITLE
[Docs] Create command ref docs and clean up other doc files

### PR DESCRIPTION
**Pull request by R7L208**
- Created docs/cmds/ file
- created commands-ref.md in docs/
- Moved command files from docs/ to cmds/
  - init.md
  - import.md
  - export.md
- created readme for docs/ and cmds/
- removed prerequisites and individual commands from main readme
  - added link to commands-ref.md in table of contents

### CODE REVIEW

File: docs/cmds/import.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+* `--destroy` - Indicates whether you wish to destroy all the provisioned infrastructure. Default is False.
+* `--plan` - Generate the Terraform plan to your infrastructure.  If set, the location will be in `<artifact-dir>/plan.out`.
+* `--apply` - Apply the plan and make modifications to the infrastructure.
+* `--revision` - Specify the git repo revision which can be a branch, commit, tag.
+
+### Example
+
+```bash
+# Import the stored state of all Databricks-native objects to the DEFAULT profile from the master branch of a remote GitHub repository
+databricks-sync import --profile DEFAULT export -g git@github.com:USERNAME/REPOSITORY.git
```
Comment (stikkireddy) [inline docs/cmds/import.md:50]: There are a few environment variables that should be documented here which is `TF_VAR_PASSIVE_MODE` and `TF_VAR_CLOUD`

File: docs/cmds/export.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+### Options
+
+* `--branch` - This is the git repo branch of the repo designated by `--git-ssh-url flag | --local-git-path`. If not given, the default branch is `master`.
+* `--ssh-key-path` or `-k` - CLI connection profile to use. The default value is `~/.ssh/id_rsa`. This is equivalent to the `-i` switch when using `ssh`.
+* `--verbosity` or `-v` - For logging, takes a value of `DEBUG`, `INFO`, `WARNING`, `ERROR`, or `CRITICAL`.
+* `--version` - A version can be attached.
+* `--config-path` or `-c` - This is the relative path (to the root directory of this repo) or the full path of the yaml file which is used to drive which objects are imported/exported.
+* `--dask` - This is a flag to use [dask](https://docs.dask.org/en/latest/) to parallelize the process.
+* `--dry-run` - This flag will log to console the actions but not commit to git remote state.
+* `--tag` - Assigns metadata to the cloud resource with [AWS tags](https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html) or [Azure tags](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/tag-support).
```
Comment (stikkireddy) [inline docs/cmds/export.md:39]: Tag is not a valid flag please remove it.

**Review by stikkireddy — CHANGES_REQUESTED** [review]

File: docs/cmds/import.md (hunk 1 above)
Comment (stikkireddy) [reply docs/cmds/import.md:50]: Maybe a section called ### Environment variables

File: docs/cmds/export.md (hunk 2 above)
Comment (R7L208) [reply docs/cmds/export.md]: Removed

File: docs/cmds/import.md (hunk 1 above)
Comment (R7L208) [reply docs/cmds/import.md]: Added a section for this.

File: docs/cmds/export.md (hunk 2 above)
Comment (R7L208) [reply docs/cmds/export.md]: Removed tag flag

**Review by stikkireddy — APPROVED** [review]

**Pull request merged**